lms/djangoapps/teams/static/teams/js/views/instructor_tools.js

```diff
@@ -26,42 +28,41 @@
                     return this;
                 },
 
-                deleteTeam: function (event) {
+                deleteTeam: function(event) {
                     event.preventDefault();
                     ViewUtils.confirmThenRunOperation(
                         gettext('Delete this team?'),
-                        gettext('Deleting a team is permanent and cannot be undone. All members are removed from the team, and team discussions can no longer be accessed.'),
+                        gettext('Deleting a team is permanent and cannot be undone. All members are removed from the team, and team discussions can no longer be accessed.'),  // jshint ignore:line
                         gettext('Delete'),
                         _.bind(this.handleDelete, this)
                     );
                 },
 
-                editMembership: function (event) {
+                editMembership: function(event) {
                     event.preventDefault();
                     Backbone.history.navigate(
-                        'teams/' + this.team.get('topic_id') + '/' + this.team.id +'/edit-team/manage-members',
+                        'teams/' + this.team.get('topic_id') + '/' + this.team.id + '/edit-team/manage-members',
                         {trigger: true}
                     );
                 },
 
-                handleDelete: function () {
+                handleDelete: function() {
                     var self = this,
-                        postDelete = function () {
+                        postDelete = function() {
                             self.teamEvents.trigger('teams:update', {
                                 action: 'delete',
                                 team: self.team
                             });
                             Backbone.history.navigate('topics/' + self.team.get('topic_id'), {trigger: true});
                             TeamUtils.showMessage(
-                                interpolate(
-                                    gettext('Team "%(team)s" successfully deleted.'),
-                                    {team: self.team.get('name')},
-                                    true
+                                HtmlUtils.interpolateHtml(
```

Comment:
By the way, not looking into the code, I was guessing that `TeamUtils.showMessage()` would use `HtmlUtils.escape()` so it could support HTML or plain text.